ModuleConfig.cfc

```diff
@@ -71,7 +71,7 @@ component {
 		//defaults
 		configStruct.autodeploy = {
 			// The tag file location, realtive or absolute
-			"tagFile" : "config/_deploy.tag",
+			"tagFile" : "#modulePath#/config/_deploy.tag",
```

Comment:
Actually Brad. The intent of the module is to have this tag file in the root application's `config` folder.